kaggle_environments/envs/connectx/connectx.json

```diff
@@ -23,7 +23,12 @@
       "default": 4,
       "minimum": 1
     },
-    "agentTimeout": 60,
+    "agentTimeout": {
```

Comment:
I've removed `agentTimeout` from the shared schema, now each environment that historically supported `agentTimeout` defines the field independently so we don't have to pollute the shared model with obsolete fields.